src/System.Runtime/tests/System/DoubleTests.netcoreapp.cs

```diff
@@ -141,6 +141,408 @@ public static void Parse_Span_Invalid(string value, NumberStyles style, IFormatP
             }
         }
 
+        /// <summary>
+        /// Test some specific floating-point literals that have been shown to be problematic
+        /// in some implementation.
+        /// </summary>
+        [Fact]
+        public static void Parse_TestTroublesomeDoubles()
+        {
+            //// https://msdn.microsoft.com/en-us/library/kfsatb94(v=vs.110).aspx
```

Comment:
Nit: what's up with the four-slash comments? :)

Reply:
This was copied as-as from the Roslyn code-base. I would ask @gafter or @jaredpar :)

Reply:
Fixed.